tests/test_hysteresis.py

```diff
@@ -0,0 +1,159 @@
+"""Test hysteresis module."""
+
+import astropy
+import mammos_entity as me
+import mammos_units as u
+import numpy as np
+import pytest
+
+from mammos_mumag.hysteresis import run
+
+
```

Comment:
Should we add tests for the multigrain case? Initial thought: Compare the extrinsic properties of a single grain cube of size 50 nm with a multigrain of the same edge length, where each grain and the non-magnetic layer has the same material parameters as the single grain. The extrinsic properties should match since the multi-grain is effectively a single grain.